test/features/lb/private_lb.feature

```diff
@@ -4,12 +4,13 @@ Feature: Test a basic tfvars configuration for the lb module with a private tier
     Scenario: Initialize testing for lb module with a private tier.
         
         Given terraform module 'lb'
-            | key      | value                     |
-            #----------|---------------------------|
-            | name     | "test-lb"                 |
-            | vpc      | "techservicesastest2-vpc" |
-            | tier     | "private"                 |
-            | internal | "true"                    |
+            | key             | value                     |
+            #-----------------|---------------------------|
+            | name            | "test-lb"                 |
+            | vpc             | "techservicesastest2-vpc" |
+            | tier            | "private"                 |
+            | internal        | "true"                    |
+            | ip_address_type | "ipv4"                    |
```

Comment:
Is this now required? Does our module have a default? Should it?

Reply:
The module does not have a default, and I don't know if it should or not. The variable description states that it should be either "ipv4" or "dualstack" so I picked one and ran with it.

Reply:
The default for v3.0.1 of the module is now `ipv4`.

Reply:
@evanjtravis since "ipv4" is now the default I recommend you drop line 13.

Reply:
I will drop the `"ipv4"` lines in the feature files.